app/src/main/java/com/bronzeswordstudios/nasaimageviewer/network/API.java

```diff
@@ -0,0 +1,59 @@
+package com.bronzeswordstudios.nasaimageviewer.network;
+
+import com.bronzeswordstudios.nasaimageviewer.model.DataResult;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface API {
+
+	@GET("search?q=galaxy&media_type=image")
+	Call<DataResult> getVariety0 ();
```

Comment:
Is there not a way to pass the search as a parameter and you do this on your end? 15 methods that are doing the same search you do below is just an overkill. You can literally use the method you have at the bottom: getSearch()

Reply:
Addressed in commit 9322ec1!